translate/rus.json

```diff
@@ -48,6 +48,7 @@
 "Benchmark over.":"Бенчмарк завершен.",
 "benchmark":"бенчмарк",
 "black mates":"мат черными",
+"black resingns":"чёрные сдались",  
```

Comment:
Lines 51 and 437 were added in case of Winboard protocol engines appearing, since they can resign on their own.

Reply:
small typo here: "resigns"

Reply:
Fixed, thanks